xbmc/music/MusicDatabase.cpp

```diff
@@ -3867,15 +3908,15 @@ void CMusicDatabase::UpdateTables(int version)
   }
   if (version < 54)
   {
-      //Remove dateAdded from artist table
-      m_pDS->exec("CREATE TABLE artist_new ( idArtist integer primary key, "
+    //Remove dateAdded from artist table
+    m_pDS->exec("CREATE TABLE artist_new ( idArtist integer primary key, "
```

Comment:
Possibly getting confused with the 2 space or 4 space tabs on different files. Trying to stick to recommened 2 space, but also be consistent with the exiting code if it is 4 space. Will review.